assets/styles/chris.scss

```diff
@@ -0,0 +1,7 @@
+$chris_blue: #0000ff;
```

Comment:
Name of variable `chris_blue` should be written in all lowercase letters with hyphens instead of underscores